src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
+});
```

Comment:
- [ ] In this store.js file, the following should have been implemented but you did not. Kindly ensure that you add the following.
   - * Imports `books` from `src/redux/books`.
   - * Imports `categories` from `src/redux/categories`.
   - * Uses `configureStore` to create a reducer.
  * `books` and `categories` should be added to `configureStore` method.

See example below 👇🏾 

```javascript
// ./src/redux/store.js

import { configureStore } from "@reduxjs/toolkit"
import booksReducer from "./books/booksSlice"
import categoriesReducer from "./categories/categoriesSlice"

const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})

export default store
```

Reply:
Thank you for giving an example.